api/pom.xml

```diff
@@ -144,8 +144,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.11.0</version>
                 <configuration>
-                    <source>11</source>
-                    <target>11</target>
+                    <source>17</source>
+                    <target>17</target>
```

Comment:
Why not remove the comment at l. 141?
Why not use the `release`?

Reply:
1. I didn't see it.
2. I was jut editing what was there.

I'll fix both of those shortly.